### ISSUE TITLE
Local Datastore

### ISSUE THREAD

**Issue by arthur-tse7** [sai-pullabhotla/catatumbo#71]
Hi @sai-pullabhotla 
Connecting to the Google Cloud Datastore is fairly easy using this library so far; is there a way to read/write to a local Datastore instance using the emulator ? I haven't managed to use the emulator using the low-level API either so I was wondering if you've had any success with it.

**Comment by sai-pullabhotla** [comment]
@arthur-tse7 - looking into this.

**Comment by sai-pullabhotla** [comment]
@arthur-tse7, 

I was able to start and work with Datastore Emulator. I'm working on providing this option from Catatumbo framework and hope to be done today (see #73). 

Follow these simple steps to get started with the low level API: 
1. Install gcloud tool 
2. Start Datastore Emulator by running the command `gcloud beta emulators datastore start --data-dir=/path/to/somedir --host-port localhost:9999`. You may be prompted to download/install additional components, answer yes to continue. Substitute the port with your preferred values. 
3. Once the Datastore starts, you should see something like this on the console: 

```
[datastore] API endpoint: http://localhost:9999
[datastore] Dev App Server is now running.
```

Now to connect to the local Datastore, use the below code: 

```
final String host="localhost";
final int port = 9999;
final String projectId = "my-project" //you can use anything that is valid for a project ID 
final String namespace = "my-namespace" //If you need namespace, otherwise, omit. 

DatastoreOptions.Builder datastoreOptionsBuilder = DatastoreOptions.newBuilder();
datastoreOptionsBuilder.setHost(host + ":" + port);
datastoreOptionsBuilder.setProjectId(projectId);
datastoreOptionsBuilder.namespace(namespace);
Datastore datastore = datastoreOptionsBuilder.build().getService();

```

**Comment by arthur-tse7** [comment]
@sai-pullabhotla 
Thanks for all the work you've done. I'll try it out once I get a shot !

**Comment by sai-pullabhotla** [comment]
This is addressed with #73. Closing.
